certbot-postfix/README.rst

```diff
@@ -2,9 +2,26 @@
 Postfix plugin for Certbot
 ==========================
 
-To install your certs with this plugin, run:
+Note: this MTA installer is in **developer beta**-- we appreciate any testing, feedback, or
+feature requests for this plugin.
 
-``certbot install --installer postfix --cert-path <path to cert> --key-path <path to key> -d <MX hostname>``
+To install this plugin, in the root of this repo, run::
 
-And there you go! If you'd like to obtain these certificates via certbot, there's more documentation on how to do this `here <https://certbot.eff.org/docs/using.html#getting-certificates-and-choosing-plugins>`_.
+    ./tools/venv.sh
+    source venv/bin/activate
+
+You can use this installer with any `authenticator plugin
+<https://certbot.eff.org/docs/using.html#getting-certificates-and-choosing-plugins>_`.
+For instance, with the `standalone authenticator
+<https://certbot.eff.org/docs/using.html#standalone>_`, which requires no extra server
+software, you might run::
+
+    certbot run --standalone -i postfix -d <domain name>
+
+To just install existing certs with this plugin, run::
+
+    certbot install -i postfix --cert-path <path to cert> --key-path <path to key> -d <domain name>
+
+If you need elevated permissions to run ``certbot``, you can run
+``sudo ./venv/bin/certbot`` with the above arguments.
```

Comment:
This is just a suggestion which you can ignore if you want, but the user will almost certainly need elevated permissions to run Certbot. We need these permissions to:

* Access Certbot's default directories
* Bind to port 80 for standalone
* Modify Postfix config files at the default path
* Run Postfix utilities

With this in mind, perhaps we should change these instructions to use `sudo` above which makes things easier for most users. There is value in giving people the option to run Certbot without root, but it requires a lot of additional setup which this doc doesn't get into.

Reply:
Good point-- Also, I'm pretty sure you need `sudo` to be able to reload `postfix` at all.